server/syntax_gen.js

```diff
@@ -0,0 +1,67 @@
+module.exports = function () {
+    var fs = require('fs');
+    var path = require('path');
+    var nodes = require('./nodetypes/nodetypes.js');
+
+    var file_path = path.join(__dirname, '../client/libs/ace-builds/src-min/snippets/fbp.js');
+
+    this.generateNodetypes = function() {
+        try {
+            var content = nodes.getNodeTypesName();
+            var arrlen = content.length;
+            var final = 'define(\"ace/snippets/fbp\",[\"require\",\"exports\",\"module\"],function(e,t,n){\"use strict\";t.snippetText=\"';
+            var triggers = [];
+            var names = [];
+
+            generateSnippets();
+
+            function generateSnippets() {
+                generateStrings();
+                for (var i = 0; i < arrlen; i++) {
+                    final = final+ 'trigger '+ triggers.pop()+ '\\n'+ 'snippet '+ names.pop()+ '\\n\\t'+ content.pop()+ '\\n';
```

Comment:
Split some things here... Also, try breaking some lines to let it be better to be read.

Example:
```
final = final + 'trigger  ' + triggers.pop() + '\\n' + 'snippet  ' + names.pop() + 
          '\\n\\t '+ content.pop() + '\\n';
```